goss.yaml

```diff
@@ -0,0 +1,22 @@
+process:
+  otelcol:
+    running: true
+
+user:
```

Comment:
I would personally only have the check on the process running + probing any http endpoint, also to be aligned with our other rocks